frontend/src/pages/StartPage/StartPage.tsx

```diff
@@ -1,5 +1,148 @@
+
+import { CACHE_KEY } from '@/constants';
+
+import { getThisDate } from '@/utils/date';
+
+import loginApi from '@/api/login';
+
+import { FcGoogle } from 'react-icons/fc';
+
+import howToUse from '../../assets/how_to_use_1.png';
+import {
+  blackTextStyle,
+  calendarStyle,
+  dateItemStyle,
+  detailTextStyle,
+  firstItemStyle,
+  firstSectionStyle,
+  googleLoginButton,
+  imageStyle,
+  introductionStyle,
+  loginText,
+  mainContentStyle,
+  methodHeaderStyle,
+  methodItemStyle,
+  methodTextStyle,
+  pageStyle,
+  refStyle,
+  secondItemStyle,
+  secondSectionStyle,
+  thirdItemStyle,
+  whiteTextStyle,
+} from './StartPage.styles';
+
 function StartPage() {
-  return <>달력에 기록을 공유할 때, 달록 입니다.</>;
+  const theme = useTheme();
+
+  const [methodAnimation, setMethodAnimation] = useState([false, false, false]);
+
+  const baseMethod = useIntersect(() => {
+    setMethodAnimation([false, false, false]);
+  });
+
+  const firstMethod = useIntersect(() => {
+    setMethodAnimation([true, false, false]);
+  });
+
+  const secondMethod = useIntersect(() => {
+    setMethodAnimation([true, true, false]);
+  });
+
+  const thirdMethod = useIntersect(() => {
+    setMethodAnimation([true, true, true]);
+  });
+
+  const { error, refetch } = useQuery<string>(CACHE_KEY.ENTER, loginApi.getUrl, {
+    enabled: false,
+    onSuccess: (data) => onSuccessGetLoginUrl(data),
+  });
+
+  const onSuccessGetLoginUrl = (loginUrl: string) => {
+    location.href = loginUrl;
+  };
+
+  const handleClickGoogleLoginButton = () => {
+    refetch();
+  };
+
+  if (error) {
+    return <>Error</>;
+  }
+
+  console.log(firstMethod, secondMethod);
+  return (
+    <PageLayout>
+      <div css={pageStyle}>
+        <section css={firstSectionStyle}>
+          <div css={calendarStyle}>
+            <div css={dateItemStyle}>{getThisDate()}</div>
+            <div css={firstItemStyle}>운동 일정</div>
+            <div css={secondItemStyle}>스터디 일정</div>
+            <div css={thirdItemStyle}>동아리 일정</div>
+          </div>
+          <div ref={baseMethod} css={refStyle} />
+
+          <div css={introductionStyle}>
+            <div css={mainContentStyle}>
+              <section>
+                <span css={blackTextStyle}>달력</span>
+                <span css={whiteTextStyle}>이</span>
+                <br />
+                <span css={blackTextStyle}>기록</span>
+                <span css={whiteTextStyle}>을</span>
+                <br />
+                <span css={whiteTextStyle}>공유할때</span>
+                <br />
+                <span css={blackTextStyle}>달록</span>
+                <br />
+              </section>
+              <span css={detailTextStyle}>
+                공유 캘린더 플랫폼 달록은 달력과 카테고리를 <br />
+                이용하여 누구나 자신의 일정을 공유할 수 있습니다.
+              </span>
+              <Button cssProp={googleLoginButton(theme)} onClick={handleClickGoogleLoginButton}>
+                <FcGoogle size={20} />
+                <p css={loginText}>Google 계정으로 계속하기</p>
```

Comment:
```suggestion
                <p css={loginText}>Google 계정으로 로그인하기</p>
```
로그인 페이지에선` 계속하기`가 적절하다고 생각했었는데 여기서는 `로그인`이라는 걸 명시해도 좋을 것 같아요~!

Reply:
아 그렇네요 그게 더 명확할 것 같아요.